package.json

```diff
@@ -132,7 +133,9 @@
     "fetch-ponyfill": "^7.1.0",
     "ipfs": "^0.65.0",
     "ipfs-geoip": "^9.0.0",
-    "typescript": "^4.8.4"
+    "typescript": "^4.8.4",
+    "workbox-build": "6.5.4",
+    "workbox-window": "6.5.4"
```

Comment:
just an observation: workbox seems to be popular on npm, but the github repo doesn't seem like it's got a ton of activity.